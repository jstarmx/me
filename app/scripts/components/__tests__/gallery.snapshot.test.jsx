const Gallery = require('../gallery');
const React = require('react');
const renderer = require('react-test-renderer');

describe('<Gallery /> (Snapshot)', () => {
  it('renders a gallery component', () => {
    const component = renderer.create(<Gallery />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
