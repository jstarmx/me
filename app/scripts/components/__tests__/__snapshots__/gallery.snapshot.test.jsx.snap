// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a Gallery component 1`] = `
<main
  className="gallery"
>
  <section
    className="gallery__section"
  >
    <h2
      className="gallery__heading"
    >
      dummy gallery
    </h2>
    <ul
      className="gallery__images"
    >
      <li
        className="thumb"
        style={
          Object {
            "backgroundImage": "url(\\"thumb\\")",
          }
        }
      >
        <a
          className="thumb__link"
          href="path"
          onClick={[Function]}
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            className="thumb__title"
          >
            title
          </span>
        </a>
      </li>
    </ul>
  </section>
</main>
`;
